packages/apps/kadena-docs/package.json

```diff
@@ -31,13 +31,14 @@
   "dependencies": {
     "@babel/plugin-syntax-import-assertions": "~7.20.0",
     "@kadena/react-components": "workspace:*",
+    "@kadena/react-ui": "workspace:*",
     "@mdx-js/loader": "~2.3.0",
     "@mdx-js/react": "~2.3.0",
     "@next/mdx": "~13.3.4",
     "acorn": "~8.8.2",
     "js-yaml": "~4.1.0",
     "mdast-util-to-string": "~3.2.0",
-    "next": "13.0.6",
+    "next": "13.4.1",
```

Comment:
🔥 
should we just let it update (at least patches)?

Reply:
yes! I'll update it 👍